lib/standard/string.nit

```diff
@@ -47,6 +47,11 @@ abstract class Text
 	#     assert "".length == 0
 	fun length: Int is abstract
 
+	# Length in bytes of the string
+	#
+	#    assert "12345".bytelen == 5
```

Comment:
You shoud add an example where the result differs with `length` (and the same example in `length`, obvioulsy)